packages/webapp/src/containers/taskSlice.js

```diff
@@ -55,6 +55,13 @@ const upsertManyTasks = (state, { payload: tasks }) => {
   );
 };
 
+const upsertOneTask = (state, { payload: task }) => {
+  state.loading = false;
+  state.error = null;
+  state.loaded = true;
+  taskAdapter.upsertOne(state, task);
+};
+
 const updateOneTask = (state, { payload: task }) => {
```

Comment:
should we remove updateOneTask if it's no longer being used?

Reply:
I don't know if it's being used elsewhere in the app